module/platform/darwin/HardwareIO/src/darwin/Darwin.cpp

```diff
@@ -78,7 +77,9 @@ namespace Darwin {
 
         // Set servos to be enabled if they are not simulated
         for (size_t i = 0; i < config["servos"].config.size(); ++i) {
-            enabledServoIds[i] = !config["servos"][i]["simulated"].as<bool>();
+            enabledServoIds[i] =
+                static_cast<__gnu_cxx::__alloc_traits<class std::allocator<unsigned char>, unsigned char>::value_type>(
+                    !config["servos"][i]["simulated"].as<bool>());
```

Comment:
This needs some justification.

Reply:
I am not sure why I didn't give this more scrutiny, as you say. I think this might be a bug. I'll post on slack.